scripts/myvectorbench.py

```
@@ -0,0 +1,641 @@
+    M = wp.get('M', 16)
+    ef = wp.get('ef_construction', 200)
+    rows = len(vectors)
+    print(f"  [index_build] {rows} rows, dim={dim}")
+
+    _create_bench_table(container, dim, rows, M, ef, "bench", "build_t")
+
+    batch = 500
+    for start in range(0, rows, batch):
+        chunk = vectors[start:start + batch]
+        vals = ", ".join(f"({start + i}, {_vec_literal(v)})" for i, v in enumerate(chunk))
+        container.sql(f"INSERT INTO bench.build_t (id, vec) VALUES {vals};")
```

Ruff (0.15.13)

[error] Possible SQL injection vector through string-based query construction

(S608)